tensorflow/python/dlpack/dlpack_test.py

```diff
@@ -16,7 +16,7 @@
 from absl.testing import parameterized
 import numpy as np
 
-
+import tensorflow as tf
```

Comment:
Never import the high level API into the internal library files. This causes an import cycle and cannot work.

Did you try to run tests for this? Any test you would have run that imported this file would have shown you the import cycle.